test/dlc_tests/sanity/test_generate_coverage_doc.py

```diff
@@ -9,8 +9,8 @@
 from test.test_utils import LOGGER, is_mainline_context
 from test.test_utils.test_reporting import get_test_coverage_file_path
 
-
-TEST_COVERAGE_REPORT_BUCKET = f"dlc-test-coverage-reports"
+account_id = boto3.client('sts').get_caller_identity().get('Account')
```

Comment:
Please leave two spaces between the import statements and the account_id assignment.

Reply:
Also nit: Please use double-quotes to comply with formatting rules we normally enforce using `black`.

Specifically, `black -l 120`.